config/base/rbac/role.yaml

```diff
@@ -4,6 +4,14 @@ kind: ClusterRole
 metadata:
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
I think we should remove this.

I would expect the base install to:
- not define this flag
- therefore not need these rules

If someone deploying OLMv1 wants to enable this feature, they can provide the appropriate (and minimal) RBAC.

Reply:
That's fair.